packages/flutter/lib/src/rendering/object.dart

```diff
@@ -1103,6 +1104,46 @@ abstract class RenderObject extends AbstractNode with DiagnosticableTreeMixin im
   /// Override in subclasses with children and call the visitor for each child.
   void visitChildren(RenderObjectVisitor visitor) { }
 
+  /// Capture an image of the current state of the render object and its
+  /// children.
+  ///
+  /// The returned [ui.Image] has uncompressed raw bytes in the dimensions of
+  /// the render object, multiplied by the [pixelRatio].
```

Comment:
We should replace "dimensions" with the actual property that we use to determine the dimensions (currently `[paintBounds]`, though if you take my feedback below, `[size]`).

Reply:
OK, size it is.